### DISCUSSION TITLE
ArcGIS Notebook Server Support

### DISCUSSION THREAD

**Discussion by WestonSF** [Esri/arcgis-powershell-dsc#548] [Q&A] [answered]
Hi there,

I see [ArcGIS Notebook Server is supported](https://github.com/Esri/arcgis-powershell-dsc/wiki/Deploy-an-ArcGIS-Notebook-Server-site-on-one-machine). Under the prerequisites it links to the Windows deployment with Linux containers. Just wondering if this is the only deployment option that is supported? or whether windows containers using the Mirantis Container Runtime would work too?

**Comment by cameronkroeker** [comment, marked as answer]
Hi @WestonSF,

Great question, thanks for asking! Yes, both Linux Containers and Windows Containers are supported with DSC. Here are some additional resources that may help:

https://github.com/Esri/arcgis-powershell-dsc/wiki/Deploy-an-ArcGIS-Notebook-Server-site-on-one-machine-that-uses-windows-containers

https://github.com/Esri/arcgis-powershell-dsc/wiki/Downloading-ArcGIS-Installation-Setups#arcgis-notebook-server-windows-container-images

Thanks,
Cameron K.
